app/js/services/sailsResponseProcessor.js

```diff
@@ -0,0 +1,35 @@
+const _ = require('lodash');
+
+function sailsResponseProcessor() {
+  const updateObject = (obj, updates) => Object.getOwnPropertyNames(updates).forEach(prop => {
```

Comment:
Great that you have unit test coverage for this, however think it would be best if you explained/ran this through with me on Monday morning.